source/releasenotes/2024-05-21-5xx-error-messages.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Unbranded 5xx error messages launched
+published_date: "2024-05-21"
+categories: [general]
+---
+
+We’ve replaced all 5xx error messages with plain HTML and removed Pantheon branding for sites hosted on Pantheon. 
+
+## Why this change
+
+Many of our customers, particularly those who are clients of our agency partners, may not be aware that their sites are hosted on Pantheon. For these clients, encountering Pantheon branded error messages could lead to confusion.
```

Comment:
```suggestion
Many of our customers, particularly those who are clients of our agency partners, may not be aware that their sites are hosted on Pantheon. For these clients, encountering Pantheon-branded error messages led to confusion.
```

Reply:
Thanks for reviewing. Good to go. @stevector  @rachelwhitton